language/tools/move-cli/README.md

```diff
@@ -75,45 +96,50 @@ a directory structure that mimics the Move memory model
 
 ### Project structure
 
-Each Move CLI project with a given `name` should have the following structure to
-it:
-
-```
-name/
-├── src # Directory containing all Move source modules
-│   ├ ...
-│   └── Module.move
-└── scripts # Directory containing all Move scripts
-    ├ ...
-    └── script.move
-```
-
-Let's now create a Move project that we'll use for the code in this README and `cd` into it:
+Each sandbox command is run in the context of a Move package. So let's create a
+Move package that we'll use for the code in this README and `cd` into it:
 
 ```shell
-$ mkdir readme
+$ move package new readme
 $ cd readme
-$ mkdir src
-$ mkdir scripts
 ```
 
 ### Compiling and running scripts
 
-Let's first start out with a simple script that prints its `signer`:
+Let's first start out with a simple script that prints its [`signer`](https://diem.github.io/move/signer.html):
 
 ```rust
 script {
 use Std::Debug;
-fun main(account: signer) {
+fun debug_script(account: signer) {
     Debug::print(&account)
 }
 }
 ```
 
-Place this in a file named `debug_script.move` under `scripts` and try
+Place this in a file named `debug_script.move` under `sources`.
+
+Before we can run this however, we need to import the Move standard library
+nursery in order to have access to the `Debug` module and `Std` [named
+address](https://diem.github.io/move/address.html#named-addresses).
+You can specify dependencies locally, or using a git URL. Here, we will specify
+it using git, so add the following to the `Move.toml` file in the `readme`
+directory:
+
+```toml
+[addresses]
+Std = "0x1" # Specify and assign 0x1 to the named address "Std"
+
+[dependencies]
+MoveNursery = { git = "https://github.com/diem/diem.git", subdir = "language/move-stdlib/nursery", rev = "d45f20a" }
```

Comment:
This is awesome, especially the `subdir` option! But one thing that's a little unclear to me when reading this document is whether a Git URL needs to be HTTP/HTTPS, or whether I can specify something like `file:///path/to/project.git`. Could you add a note explaining what's allowed?